test/banners/desktop/components/BannerVar.spec.ts

```diff
@@ -10,11 +10,11 @@ import { TrackerStub } from '@test/fixtures/TrackerStub';
 import { softCloseFeatures } from '@test/features/SoftCloseDesktop';
 import { useOfFundsFeatures } from '@test/features/UseOfFunds';
 import {
-	bannerContentAnimatedTextFeatures, bannerContentDateAndTimeFeatures,
```

Comment:
I don't get why you're removing the tests for `bannerContentDateAndTimeFeatures` - is it because it's a regular feature of banners now that was tested once?

Reply:
Whoops, copy and paste error